### PR TITLE
Raise version to 7.5.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by sutaakar**
To be in sync with master.

### CODE REVIEW

File: framework-cloud/cloud-deployment-plugin/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>org.kie.cloud</groupId>
     <artifactId>framework-cloud</artifactId>
-    <version>7.4.1-SNAPSHOT</version>
+    <version>7.5.0-SNAPSHOT</version>
```
Comment (jhrcek) [inline framework-cloud/cloud-deployment-plugin/pom.xml]: Is it really necessary to have this in all the poms? Wouldn't using `${project.parent.version}` be better?

**Comment by sutaakar** [comment]
test this

**Comment by sutaakar** [comment]
test this

File: framework-cloud/cloud-deployment-plugin/pom.xml (hunk 1 above)
Comment (sutaakar) [reply framework-cloud/cloud-deployment-plugin/pom.xml]: @jhrcek Unfortunately no, Maven isn't able to resolve expression in parent version.

**Pull request merged**